Guide/src/dev_guide/contrib/release.md

```diff
@@ -0,0 +1,41 @@
+# Release Management
+
+ Occasionally, the OpenVMM project will declare upcoming release milestones. We stabilize the code base in a `release/YYMM` branch, typically named for the YYMM when the branch was forked. We expect a high quality bar for all code that goes in to the OpenVMM main branch, we ask developers to hold these `release/YYMM` to the highest quality standards. The OpenVMM maintainers will gradually slow the rate of churn into these branches as we get closer to a close date.
+
+ This process should not impact your typical workflow; all new work should go into the `main` branch. But, to ease the cherry-picks, we may ask that you hold off from making breaking or large refactoring changes at points in this process.
+
+## Marking, Approval Process, Code Flow
+
+The OpenVMM maintainers will publish various dates for the upcoming releases. Currently, these dates are driven by a Microsoft-internal process and can, and do, often change. Microsoft does not mean to convey any new product launches by choices of these dates.
```

Comment:
Do we want to talk about the rough cadence we're aiming for between releases?

Reply:
We could, but I don't think we have enough data to describe that at the moment.